src/task2.ts

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs';
+import csv from 'csvtojson';
+import CSVError from 'csvtojson/v2/CSVError';
+
+const CSV_FILE = './src/nodejs-hw1-ex1.csv';
```

Comment:
Maybe usefull to not set path relative to project folder, but to the current work directory
For example `path.join(__dirname, fileName);`
Or, maybe, you'll go further and pass file name as argument like `node taks2.ts nodejs-hw1-ex1` and then in the code `const CSV_FILE = path.join(__dirname, '${process.argv[2]}.csv';`
`const OUTPUT_FILE = path.join(__dirname, '${process.argv[2]}.txt';`

But of course it is not critical and up to you - if you don't like this approach feel free to remain it as is 🙂

Reply:
@mccher yeah, sure. i was thinking about the second improvement (file names from arguments) also i find good to use current work directory. i will add both 😄

Reply:
@mccher I also added logic related to "writing" a chunk to db and after a success writing it into a file